lighthouse-core/report/report-generator.js

```diff
@@ -132,7 +132,14 @@ class ReportGenerator {
       hour: 'numeric', minute: 'numeric', second: 'numeric',
       timeZoneName: 'short'
     };
-    const formatter = new Intl.DateTimeFormat('en-US', options);
+    let formatter = new Intl.DateTimeFormat('en-US', options);
+
+    // Force UTC if runtime timezone could not be detected.
+    // See https://github.com/GoogleChrome/lighthouse/issues/1056
+    if (formatter.resolvedOptions().timeZone.match(/Unknown/i)) {
```

Comment:
I think you should still check for `"Etc"`. That's an [important part of the zone name](https://en.wikipedia.org/wiki/Tz_database#Area), I just learned, and I'm not entirely sure that you can't have an `Unknown` location inside other Areas.